.env.production

```
@@ -8,4 +8,4 @@ NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=926871086070
 NEXT_PUBLIC_FIREBASE_APP_ID=1:926871086070:web:9a01bad8170cb172f15dd7
 
 # Keep empty in Firebase Hosting builds so /api/** uses firebase.json rewrites.
-NEXT_PUBLIC_BACKEND_API_BASE_URL=
+NEXT_PUBLIC_BACKEND_API_BASE_URL= https://aim-be-prod-926871086070.us-central1.run.app
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)